src/Elmish.WPF/Binding.fs

```diff
@@ -1263,6 +1263,22 @@ type Binding private () =
     >> Binding.mapModel get
     >> Binding.mapMsgWithModel set
 
+  /// <summary>Creates a two-way binding.</summary>
+  /// <param name="get">Gets the value from the model.</param>
+  /// <param name="set">Returns the message to dispatch.</param>
+  /// <param name="wrapDispatch">
+  ///   Wraps the dispatch function with additional behavior, such as
+  ///   throttling, debouncing, or limiting.
+  /// </param>
+  [<System.Obsolete("In version 5, this method will be removed.  Use the overload without the \"wrapDispatch\" parameter followed by a call to \"Binding.alterMsgStream\".  For an example, see how this method is implemented.")>]
+  static member twoWay
```

Comment:
Add an obsolete overload with the `wrapDispatch` parameter and call the non-obsolete overload followed by the call to `Binding.alterMsgStream wrapDispatch`.

Reply:
I totally missed this edge case. I have tried a few things and I can't find any good way to solve this so yeah +1 on saying let's have a potential breaking change edge case.

Reply:
Actually, we could also add an overload that accepts `Option<>`.  Maybe this idea will help in the more complicated cases below.

Reply:
> I think the only edge case is if the user explicitly passed in an `Option<>`. I think that is possible in `v3` (thought I could be wrong),

I just tested; I am wrong.  Passing in `Option<>` for an optional parameter is not possible.